var/spack/repos/builtin/packages/py-pennylane/package.py

```diff
@@ -12,15 +12,16 @@ class PyPennylane(PythonPackage):
 
     homepage = "https://docs.pennylane.ai/"
     git = "https://github.com/PennyLaneAI/pennylane.git"
-    pypi = "PennyLane/PennyLane-0.28.0.tar.gz"
+    url = "https://github.com/PennyLaneAI/pennylane/archive/refs/tags/v0.29.0.tar.gz"
 
     maintainers("mlxd", "AmintorDusko", "marcodelapierre")
 
     version("master", branch="master")
-    version("0.29.1", sha256="c5d662994b741afa69e4fdadc79a1b75840275138a8b7e0cfc5fd64b66a12eef")
+    version("0.29.1", sha256="6ecfb305a3898347df8c539a89a67e748766941d159dbef9e34864872f13c45c")
+    version("0.29.0", sha256="e59e042174abd8f1596034d3f4dd113b24d06e4aadb5f2e5828fa3411fabf0df")
```

Comment:
Because we have the 0.29.1 bug-fix release, we prefer not to add version 0.29.0, since it is already deprecated.

Reply:
Do you prefer removing such versions or marking them `deprecated=True`?

Reply:
In this case, the best course of action is to just remove it.